sdks/python/test-suites/tox/py39/build.gradle

```diff
@@ -154,17 +154,17 @@ test.dependsOn "testPy39tensorflow-212"
 postCommitPyDep.dependsOn "testPy39tensorflow-212"
 
 // Create a test task for each minor version of transformers
-toxTask "testPy39transformers-428", "py39-transformers-428", "${posargs}"
-test.dependsOn "testPy39transformers-428"
-postCommitPyDep.dependsOn "testPy39transformers-428"
+toxTask "testPy39transformers-445", "py39-transformers-445", "${posargs}"
+test.dependsOn "testPy39transformers-445"
+postCommitPyDep.dependsOn "testPy39transformers-445"
 
-toxTask "testPy39transformers-429", "py39-transformers-429", "${posargs}"
-test.dependsOn "testPy39transformers-429"
-postCommitPyDep.dependsOn "testPy39transformers-429"
+toxTask "testPy39transformers-446", "py39-transformers-446", "${posargs}"
+test.dependsOn "testPy39transformers-446"
+postCommitPyDep.dependsOn "testPy39transformers-446"
 
-toxTask "testPy39transformers-430", "py39-transformers-430", "${posargs}"
-test.dependsOn "testPy39transformers-430"
-postCommitPyDep.dependsOn "testPy39transformers-430"
+toxTask "testPy39transformers-447", "py39-transformers-447", "${posargs}"
+test.dependsOn "testPy39transformers-447"
+postCommitPyDep.dependsOn "testPy39transformers-447"
```

Comment:
Do you know what the underlying issue is with these dependencies? It would be better to keep one of these as a lower bound if possible so that we are able to test that beam is still compatible with lower versions of transformers. My ideal would be to have 2 tests, one that is `testPy39transformers-428` to test our lower bound and one that is `testPy39transformers-447` to test our upper bound

Reply:
@damccorm The tests testPy39transformers-428, 429, 430 were using the latest version of sentence-transformers (3.4.0), which doesn't support these transformers versions. As a result, it pulled the latest transformers 4.48.1, which no longer supports older torch (<2.0.0) versions that we were using, causing the tests to fail.
So I kept tests for:
- transformers 4.28.x (with sentence-transformers 2.2.2, the latest version that supports 4.28)
- transformers 4.47.x (the last version that still supports older torch)
- transformers 4.48.x (the latest overall version, uses upgraded torch).